spec/test/stored_queries/test_fulltext_search.py

```diff
@@ -0,0 +1,226 @@
+"""g
```

Comment:
`g`?

Reply:
No idea ... maybe I was in edit mode in vim when I thought I was in navigation mode